ui/components/src/model/theme.ts

```diff
@@ -20,6 +20,17 @@ export interface PersesChartsTheme {
     width: number;
     color: string;
   };
+  /**
+   * Theming for the container that wraps a chart.
+   */
+  container: {
+    /**
+     * Padding in pixels.
+     */
+    padding: {
+      default: number;
```

Comment:
Intentionally including a `default` to start here, so we can potentially specify other spacing in the future without introducing a breaking change in external plugins that depend on the value. e.g. I could imagine us specifying a "small" value we want to use for charts that fit a little more tightly.